tests/spring-service/simple-service/expected/spring-service/templates/pre-deployment/secret.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Source: spring-service/templates/pre-deployment/secret.yaml
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  namespace: team-supercool
+  name: myservice-external-secret
+spec:
+  backendType: systemManager
+  roleArn: arn:aws:iam::1234567890:role/read-secrets-role-staging-team-superpower
```

Comment:
Die ARN ist ja jetzt leider schon relativ weit verbreitet, mir persönlich hätte aber eher sowas wie `k8s-<env>-secrets-ns-<ns>` gefallen. Hier kommt weder `-ns-` vor, noch `k8s` und `role` ist überflüssig, das steht in der ARN und definiert das ganz klar. User heißen auch nicht `user-` :-)